product-development/README.md

```diff
@@ -0,0 +1,83 @@
+When a Tilt engineer is free, they should pick a high priority epic to work on and make themself the _sole_ owner of the epic. The engineer should find other folks to collaborate on the epic as needed, but they are the [DRI](https://medium.com/@mmamet/directly-responsible-individuals-f5009f465da4) (directly responsible individual) for driving the epic to completion. There should be at most one owner on any epic, and the owner is the DRI.
+
+The epic may have a fairly broad initial business problem. The engineer(s) should carve out and define a smaller problem, and spec out a solution that can be accomplished within a target of **_7 working days_** (counted from when the epic moves into `In Progress`, more below on states). If the solution cannot be finished within 7 days, keep reducing scope. The epic description should be updated with the smaller scope and acceptance criteria, and the engineer(s) should create additional epics and/or stories to capture the future work.
+
+The purpose of a single epic is _not_ to completely solve a business problem. Rather, completing an epic should likely achieve incremental progress at addressing a single problem, adding incremental user value as a result. In some cases, any realized user value may only appear in future epics. 
+
+After the epic is completed, the engineer should find the next epic to work on, either as a DRI owner, or as a collaborator. Engineers should self-organize and coordinate timelines ad hoc when picking epics. There are no stable teams, no timeboxed sprints, and no synced sprints across teams. Engineers should avoid working on multiple epics concurrently. 
+
+### Epic owner and epic states
+- An epic is in one of three states: `Unstarted`, `In Progress`, `Closed`. 
+- If an epic is in `In Progress` or `Closed`, it must have exactly one owner, who is the DRI.
+
+### DRI responsibilties and non-responsibilities as epic owner
+
+| What | Reponsibilities | Non-responsibilities | Why |
+| --- | --- | --- | --- |
+| Communication and status | The DRI is responsible for maintaining the epic state, description, and attached stories updated as the single source of truth of overall work status, at least on a daily basis.<br/><br/>The acceptance criteria sections are _especially critical_, including presenting the problem and final implementation in a company meeting.| The DRI is not responsible for communicating epic work status, beyond keeping the epic itself updated as the single source of truth. The DRI is not responsible for sharing work on Slack, meetings, or keeping Tilters otherwise updated. The DRI _can_ provide additional communication, but is not required to do so. Victor will help with broader general communication needs. | This encourages anyone who wants to learn about and collaborate on the epic to visit the epic itself (and it's attached stories) to see the latest updates, allowing the DRI to focus on actual work and leverage a single place for providing updates. |
```

Comment:
> Do you expect all Tilters to have a common understanding of the right level of detail in the epic state / story state? Or do you expect each Tilter to maintain their stories at different levels of granularity?

When I say "epic state", I mean Unscheduled/In Progress/Closed. And the description of an epic has the acceptance criteria. So I expect that to be clear since people are just following a specific format. In terms of how people split out work in stories, and how to generally use stories, I purposely didn't introduce this in this PR, since I didn't want to add more process and overwhelm people all in one shot. 

> When you say "encourages anyone who wants to learn about and collaborate on the epic [can] visit the epic itself", can you describe what you expect that interaction to look like? Do you expect the majority of this communication to be:

Yes to all of these. If someone is interested in a particular epic that is in progress, I want them to be able to jump in and opine and start a discussion, and even offer to help out. But ultimately the DRI gets to decide what to do with that discussion. I expect the DRI to be sensible and see great feedback and be able to make good judgment calls on whether to take action on the feedback, or defer it to future work.